zetaclient/tx.go

```diff
@@ -301,3 +308,40 @@ func (b *ZetaCoreBridge) PostAddBlockHeader(chainID int64, blockHash []byte, hei
 	}
 	return "", fmt.Errorf("post add block header failed after %d retries", DefaultRetryCount)
 }
+
+// MonitorTxResult monitors the result of a tx (used for inbound and outbound vote txs)
+func (b *ZetaCoreBridge) MonitorTxResult(zetaTxHash string, isInbound bool) {
+	var lastErr error
+	ticker := 5 * time.Second
+	retry := 10
```

Comment:
Maybe we can move these values to the Config struct .  Wdyt
Does not have to be pr , we can just create an issue for now